JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -12,7 +12,9 @@
     localDocument.getElementsByTagName(scriptText)[0].parentNode.appendChild(scriptElement);
 
     // capture initial cookie
-    appInsights.cookie = localDocument.cookie;
+    try {
+        appInsights.cookie = localDocument.cookie;
```

Comment:
Just wondering if there's a corresponding code that expects/does not expect appInsights.cookie to be unset